gdtoolkit/parser/gdscript.lark

```diff
@@ -255,18 +255,8 @@ _lambda_suite: _lambda_body
              | _standalone_lambda_stmt
 _lambda_body: _NL _INDENT _func_stmt+ _DEDENT
 _standalone_lambda_stmt: _simple_func_stmt
-                       // | annotation* compound_lambda_stmt
```

Comment:
opinionated, but I think fixing compound func stmt is the way to go and this works *some* of the time. could def revert this part tho